pacman/operations/rigged_algorithms/isomorph_check.py

```diff
@@ -67,4 +77,6 @@ def check(self, placements, placements_copy):
                         placements_copy.get_vertex_on_processor(x, y, p))
 
         # if the two sets are not
-        return chip_vertices != chip_vertices_copy
+        return (chip_vertices != chip_vertices_copy,
```

Comment:
Although I note that this matches the old version, should this be chip_vertices == chip_vertices_copy?